GMC_identification.ipynb

```diff
@@ -0,0 +1,1603 @@
+{
```

Comment:
<p>Major: module not found error 'no module named scimes' -- cannot proceed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Smithsonian/sma-science-tutorials/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>noted. to be fixed (needs to a bug fix in scimes itself)</p>